swift/src/Ice/Proxy.swift

```diff
@@ -315,21 +315,20 @@ public func makeProxy(communicator: Ice.Communicator, proxyString: String, type:
     try communicator.makeProxyImpl(proxyString) as ObjectPrxI
 }
 
-/// Casts a proxy to `Ice.ObjectPrx`. This call contacts the server and will throw an Ice run-time exception
-/// if the target object does not exist or the server cannot be reached.
+/// Creates a new proxy from an existing proxy after confirming the target object's type via a remote invocation.
 ///
-/// - parameter prx: `Ice.ObjectPrx` - The proxy to cast to `Ice.ObjectPrx`.
+/// - parameter prx: `Ice.ObjectPrx` - The source proxy.
```

Comment:
Note that the Swift doc-comments format still needs to be updated. They are currently using an odd mix of Python/MATLAB doc-comment style with Swift flavoring.

Reply:
There's an open issue tracking this: #2274.
It's fixed in the generated code, but the hand-written code still needs fixing.